File: 프로그래머스/0/120888. 중복된 문자 제거/중복된 문자 제거.cs
using System;

public class Solution 
{
    public string solution(string my_string) 
    {
        string str = "";
        for(int i=0; i<my_string.Length; i++) {
            if(!str.Contains(my_string[i])) {
                str += my_string[i].ToString();
            } 
        }
        return str;
    }
}

File: 프로그래머스/0/120888. 중복된 문자 제거/중복된 문자 제거.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string my_string) 
{
    string temp = "";
    for(char &c : my_string)
    {
        if(find(temp.begin(), temp.end(), c) != temp.end())
        {
            continue;
        }
        else
        {
            temp += c;
        }
    }
    return temp;
}